### PR TITLE
IHRACP-5801: Pubdate Investigation

### PR DESCRIPTION
**Pull request by poonamshinde-ihm**
## JIRA Ticket
- [Pubdate Investigation](https://ihm-it.atlassian.net/browse/IHRACP-5801)

#### Major Changes:
- Added PDT Timezone in pytz timezon for PDT to EST conversion
- unit test added for PDT to EST conversion

## Related PRs

- https://github.com/iheartradio/episodeingestion/pull/41

### CODE REVIEW

**Review by IanNoyes — APPROVED** [review]
Tested this in whisper with a PDT timezone and EST timezone.  The PDT timezone episodes were converted to EST as expected and the EST timezones imported as normal.  Good work!

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 setup(
     name='pypodcastparser-ihr',
 
-    version='1.8.0',
+    version='1.9.0.dev',
```
Comment (IanNoyes) [inline setup.py]: I lied, let's change this version number to not include the dev tag.  After that we're good.

**Review by IanNoyes — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (poonamshinde-ihm) [reply setup.py]: sure Thanks

Comment (poonamshinde-ihm) [reply setup.py]: done

**Review by IanNoyes — APPROVED** [review]

File: pypodcastparser/Item.py [hunk 2]
```diff
@@ -26,7 +26,8 @@
 'PST' : 'US/Pacific' ,
 'CST' : 'US/Central' ,
 'CAT' : 'Africa/Maputo' ,
-'AEST' : 'Australia/Sydney'
+'AEST' : 'Australia/Sydney',
+'PDT' : 'America/Los_Angeles'
```
Comment (daleyp24) [inline pypodcastparser/Item.py]: have we investigated in kibana any other failing timezones or asked nate

**Comment by daleyp24** [comment]
need to address linter errors in tox

**Review by daleyp24 — CHANGES_REQUESTED** [review]
see comments

**Comment by poonamshinde-ihm** [comment]
> need to address linter errors in tox

fixed

**Review by daleyp24 — APPROVED** [review]

**Pull request merged**